### PR TITLE
Adds drains under ChemiCompilers

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
Adds drains under ChemiCompilers

## Why's this needed? 
This comes from a feature request from someone (I forgot who, Im sorry).
The CC has a function to dump it's contents out, which spills it out on the floor. Someone asked if drains could be added onto the floor so that it doesnt keep making a mess.

### CODE REVIEW

File: maps/clarion.dmm [hunk 1]
```diff
@@ -33278,6 +33278,7 @@
 	dir = 1;
 	pixel_y = 30
 	},
+/obj/machinery/drainage,
```
Comment (Azrun) [inline maps/clarion.dmm:33281]: And a voice rang out, "No changes to Clarion are currently permitted".

**Comment by Azrun** [comment]
Should the drain be adjacent to the ChemiCompiler instead?  Seems like you could use the ChemFuck to time and mix/release a hellchem that you want to fill the room.  You would be able to `ctrl+alt+shift` and clog the drain, but it would improbable for anyone else to realize that is what happened or how to deal with it (except for 🤓).

**Comment by ZeWaka** [comment]
yeah i don't like having the drain be under the machine, that limits mechanics like flooding the room

drains adjacent would be fine though

**Review by ZeWaka — CHANGES_REQUESTED** [review]
x

**Comment by Azrun** [comment]
Looks like some of the drains directly under the ChemiCompilers are still there. (As per Map Diff Render of 0ef6dc9)

Kondaru
kondaru

Horizon
horizon

Oshan
image

**Review by ZeWaka — CHANGES_REQUESTED** [review]
X

**Comment by geneva-breaker** [comment]
Not a huge fan. 
This is one of those "If the mapper intended to do this, they would have done this" situations in my opinion. One of the CC's strenghths is its use in flooding and foaming on the floor, and if someone wanted to install a drain I think they could do that in-round. 

If it's a common-enough irritant, maybe providing chemistry departments with One ready-to-deploy drain in a box would be a good QoL improvement, they could use it wherever they like.

**Pull request closed without merging**